var/spack/repos/builtin/packages/mesa/package.py

```diff
@@ -236,12 +283,36 @@ def osmesa_libs(self):
 
     @property
     def glx_libs(self):
+        if '+glvnd' in self.spec:
```

Comment:
Can we drop the early exit
```python
if '+glvnd' in self.spec:
    return LibraryList()
```
from the `glx_libs`, `egl_libs`, and `gl_libs` properties since the property functions should never end up getting invoked anyways when `+glvnd` is specified?